ext2/ext2.c

```diff
@@ -898,7 +971,8 @@ static int _load_file(
 {
     int ret = 0;
     struct stat st;
-    char block[1024];
+    char* block = NULL;
+    const size_t block_size = 1024;
```

Comment:
Make it `MYST_BLKSIZE`.

Reply:
``MYST_BLKSIZE`` is 512 and I am afraid to change it. I am just trying to preserve existing behavior for now.